testing/src/main/java/io/grpc/internal/testing/AbstractTransportTest.java

```diff
@@ -471,20 +535,31 @@ public void ping_afterTermination() throws Exception {
 
   @Test
   public void newStream_duringShutdown() throws Exception {
```

Comment:
I'm not excited about how many places stats are checked in these tests as well. I don't know why we would need so many asserts for stats in newStream_duringShutdown, for example.